src/olympia/files/forms.py

```diff
@@ -38,8 +38,7 @@ def render_options(self, context):
         def option(files, label=None, deleted=False, channel=None):
             # Make sure that if there's a non-disabled version,
             # that's the one we use for the ID.
-            files.sort(lambda a, b: ((a.status == amo.STATUS_DISABLED) -
-                                     (b.status == amo.STATUS_DISABLED)))
+            sorted(files, key=lambda a: a.status == amo.STATUS_DISABLED)
```

Comment:
Wait, that won't work… `sorted` doesn't do the sorting inline but returns the sorted result.

Reply:
Ahah I'm dumb :) I wanted to return it but forgot. Funny enough the test wasn't right and I changed it... I guess it's still not right since it's passing.

Reply:
This is fixed in https://github.com/mozilla/addons-server/pull/10589/commits/1b8f1cd72ee2f1ff0188df07b21ab0944baeb0d5